test/parallel/test-worker-prof.js

```diff
@@ -32,10 +32,11 @@ for (const logfile of logfiles) {
   const lines = fs.readFileSync(logfile, 'utf8').split('\n');
   const ticks = lines.filter((line) => /^tick,/.test(line)).length;
 
-  // Test that at least 20 ticks have been recorded for both parent and child
+  // Test that at least 15 ticks have been recorded for both parent and child
   // threads. When not tracking Worker threads, only 1 or 2 ticks would
   // have been recorded.
-  // When running locally on x64 Linux, this number is usually at least 150
+  // When running locally on x64 Linux, this number is usually at least 700
   // for both threads, so 15 seems like a very safe threshold.
+  console.log(ticks)
```

Comment:
Isn't the assertion message sufficient?

Reply:
This may have been a leftover from debugging :)